optuna_dashboard/ts/components/GraphSlice.tsx

```diff
@@ -187,6 +187,8 @@ const plotSlice = (
     return t.params.find((p) => p.name == selected)!.value
   })
 
+  const trialNumbers: number[] = filteredTrials.map((t) => t.number as number)
```

Comment:
[nits] Can we remove `as number` here?

```suggestion
  const trialNumbers: number[] = filteredTrials.map((t) => t.number)
```